org.gnome.gThumb.yml

```diff
@@ -32,13 +32,13 @@ modules:
       - /lib/exiv2
       - /share/man
     config-opts:
-      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
Is this useful?

Reply:
Debug info get generated (and packaged separately). It's usefull to gather stack trace when needed and it doesn't impact otherwise.